names/names.py

```diff
@@ -16,6 +16,17 @@
         if name_1 == name_2:
             duplicates.append(name_1)
 
+"""
+The two are my solutions: both are O(n lon n), it takes about 1.40 secs on my machine
+"""
+duplicates = []
+for name in names_1:
+    if name in names_2:
+        duplicates.append(name)
```

Comment:
I think that these two solutions are still not as optimized as they could have been. You could have made one of the names lists into a set which would make the solution have O(n) time complexity. Or if you used a BST you could have a solution of O(n log n). I'm pretty sure a n log n solution would have a faster sort that 1.40 sec because the modal solution has a runtime of .17 secs